net/xtables-addons/patches/002-restore-support-for-Linux-4.14.patch

```diff
@@ -34,11 +30,9 @@ index d08354a..faf5dd8 100644
  #endif
  
  #if defined(CONFIG_NF_CONNTRACK) || defined(CONFIG_NF_CONNTRACK_MODULE)
-diff --git a/extensions/pknock/xt_pknock.c b/extensions/pknock/xt_pknock.c
-index c76901a..7be0345 100644
 --- a/extensions/pknock/xt_pknock.c
 +++ b/extensions/pknock/xt_pknock.c
-@@ -357,10 +357,18 @@ has_logged_during_this_minute(const struct peer *peer)
+@@ -356,10 +356,18 @@ has_logged_during_this_minute(const stru
```

Comment:
This has always struck me as amusing.

Do you know what is responsible for this?

Reply:
What do you mean?

Reply:
git diff creates those long lines. GNU diff (or whatever it it) shortens them. No idea why though.

Reply:
It is the 'quilt refresh' process that does the reduction in context, the stripping of index and stripping of comments from the non-topmost patch in a patch file containing multiple patches.

It is advisable to run 'make package/foo/refresh' when updating packages/patches to reduce the level of patch format churn and catch any fuzz.

Ensuring .quiltrc is setup correctly is a good idea too - https://openwrt.org/docs/guide-developer/build-system/use-patches-with-buildsystem